web_site/domain/user.py

```diff
@@ -0,0 +1,90 @@
+##############################################################################
+### user.py - User model                                                   ###
+##############################################################################
+#                                                                            #
+#   Description : Defines users of the application                           #
+#                                                                            #
+#   External Calls:                                                          #
+#                                                                            #
+#                                                                            #
+#  History                                                                   #
+#                                                                            #
+#  Name       Date               Update                                      #
+# =======    ==========         ========================================     #
+# Jerry      08/21/2016         Module created                               #
+#                                                                            #
+##############################################################################
```

Comment:
Haha no probs. Its a habit moreso to keep track of changes than for
ownership

On Aug 21, 2016 8:35 PM, "Frito" <notifications@github.com> wrote:

> In web_site/domain/user.py
> <https://github.com/MadDonkeySoftware/pyDuesPal/pull/3#discussion_r75607135>
> :
>
> > +##############################################################################
> > +### user.py - User model                                                   ###
> > +##############################################################################
> > +#                                                                            #
> > +#   Description : Defines users of the application                           #
> > +#                                                                            #
> > +#   External Calls:                                                          #
> > +#                                                                            #
> > +#                                                                            #
> > +#  History                                                                   #
> > +#                                                                            #
> > +#  Name       Date               Update                                      #
> > +# =======    ==========         ========================================     #
> > +# Jerry      08/21/2016         Module created                               #
> > +#                                                                            #
> > +##############################################################################
>
> @rolltide0 <https://github.com/rolltide0> Yea, we don't own code here. Go
> ahead and delete this section. I personally feel the "headings" through the
> file are needless as well so if the group agrees we'd want to delete those
> as well.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/MadDonkeySoftware/pyDuesPal/pull/3/files/03ddbea22fd6c103dbb4ec55d82e54a8fa3d41f1#r75607135>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AI56IVw622FI9dnt69vvt-GGS6L8XRo6ks5qiO7RgaJpZM4JpW2K>
> .
>